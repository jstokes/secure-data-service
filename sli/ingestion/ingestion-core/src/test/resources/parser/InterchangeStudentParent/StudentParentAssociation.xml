<?xml version="1.0" encoding="UTF-8"?>
<InterchangeStudentParent xmlns:sli="http://slc-sli/ed-org/0.1" xmlns="http://ed-fi.org/0100"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
    <StudentParentAssociation>
        <StudentReference>
            <StudentIdentity>
                <StudentUniqueStateId>_hvNB7n1juH</StudentUniqueStateId>
            </StudentIdentity>
        </StudentReference>
        <ParentReference>
            <ParentIdentity>
                <ParentUniqueStateId>aBH75KvXygxZ</ParentUniqueStateId>
            </ParentIdentity>
        </ParentReference>
        <Relation>Cousin</Relation>
        <PrimaryContactStatus>false</PrimaryContactStatus>
        <LivesWith>true</LivesWith>
        <EmergencyContactStatus>true</EmergencyContactStatus>
        <ContactPriority>-742574136</ContactPriority>
        <ContactRestrictions>H.z1D2bJ0PqkooeIJ05nPHywA5BBT.N1cXcFcgWOpV</ContactRestrictions>
    </StudentParentAssociation>
</InterchangeStudentParent>
