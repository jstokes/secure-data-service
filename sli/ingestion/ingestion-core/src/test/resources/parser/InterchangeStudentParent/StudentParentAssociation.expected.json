{
  "StudentReference":{
    "StudentIdentity":{
      "StudentUniqueStateId":{
        "_value":"_hvNB7n1juH"
      }
    }
  },
  "ParentReference":{
    "ParentIdentity":{
      "ParentUniqueStateId":{
        "_value":"aBH75KvXygxZ"
      }
    }
  },
  "Relation":{
    "_value":"Cousin"
  },
  "PrimaryContactStatus":{
    "_value":false
  },
  "LivesWith":{
    "_value":true
  },
  "EmergencyContactStatus":{
    "_value":true
  },
  "ContactPriority":{
    "_value":-742574136
  },
  "ContactRestrictions":{
    "_value":"H.z1D2bJ0PqkooeIJ05nPHywA5BBT.N1cXcFcgWOpV"
  }
}
